template/scripts/docs_templating.sh

```diff
@@ -30,9 +30,10 @@ then
   echo "$templating_vars_file does not exist, cannot start templating."
 fi
 
-for file in $(find "$docs_dir" -name '*.j2')
+find "$docs_dir" -name '*.j2' |
+while read -r file
 do
-  new_file_name=$(echo "$file" | sed 's/.j2$//')  # cut of the '.j2'
+  new_file_name=${file//.j2/}
```

Comment:
This solution was discussed, but it isn't identical to the `sed` variant. the new version will replace *all* .j2 occurrences. I ignored the linter in that case, because it was only a suggestion.

Reply:
Correct! Updated to only remove the suffix